src/modules/uORB/Subscription.hpp

```diff
@@ -84,6 +84,38 @@ class Subscription
 		subscribe();
 	}
 
+	// Copy constructor
+	Subscription(const Subscription &other)
```

Comment:
You can directly initialize them:
```suggestion
	Subscription(const Subscription &other) : _orb_id(other._orb_id), _instance(other._instance)
```

Reply:
Thanks, done.